docs/source/api/python_library.rst

```diff
@@ -1,6 +1,76 @@
 Python library
 ==============
 
-This section only contains code reference (autogenerated docs are possible with a description of arguments and return values)
 
-.. Attention:: Contents are missing for now.
\ No newline at end of file
+About Nexus
+---------------
+https://nexus.soramitsu.co.jp/ - our new PyPI, Maven, artifacts server with precompilled librares.
+You can download binaries without compilation on the client side. It helps to save you time and disk space for installing dependencies like libboost, libz, etc.
+
+Now our packages are distributed only for ``Windows x86_64`` with ``Python 2.7.x`` and ``Python 3.5.x``. In future it will be available on ``manylinux`` platform and ``Mac OS``.
+
+https://nexus.soramitsu.co.jp/repository/pypi-dev/ - developer wheels
+https://nexus.soramitsu.co.jp/repository/pypi-release/ - release wheels
+https://nexus.soramitsu.co.jp/repository/pypi-dev-nightly/ - dev nightly wheels
+
+
+Pulling from PyPI
+---------------
+
+You can download our PyPI package from our Nexus server (instead of pypi.org). To do this, please specify ``-i`` parameter:
+
+.. code-block:: shell
+
+    pip install -i https://nexus.soramitsu.co.jp/repository/pypi-{develop,release,develop-nightly}/ iroha 
```

Comment:
Please make it directly copyable without any modifications. E.g., `pip install -i https://nexus.soramitsu.co.jp/repository/pypi-develop/ iroha`. I suggest to split it into three lines with annotations on what each repository consists of (i.e., `developer` build from `develop` branch, etc.)